src/pages/home/sidebar/OptionRow.js

```diff
@@ -194,15 +194,15 @@ const OptionRow = ({
                         <View style={[styles.flexRow, styles.alignItemsCenter]}>
                             {option.hasDraftComment && (
                                 <View style={styles.ml2}>
-                                    <Icon src={Pencil} height="16" width="16" />
+                                    <Icon src={Pencil} height={16} width={16} />
```

Comment:
Thanks for fixing this :D It must have been causing a propType warning? I'm ashamed I didn't notice that

Reply:
Oh yes forgot to mention that I fixed a few random `propTypes` things in this.